svc-bie-kafka/src/main/java/gov/va/vro/services/bie/config/MessageQueueConfig.java

```diff
@@ -34,7 +35,7 @@ Declarables topicBindings(final BieProperties bieProperties) {
         bieProperties.getKafkaTopicToAmqpQueueMap().values().stream()
             .map(
                 topic -> {
-                  final Queue queue = new Queue(topic, DURABLE);
+                  final Queue queue = new Queue(topic, DURABLE, EXCLUSIVE, AUTO_DELETE);
```

Comment:
This use of constantants burdens the reader to lookup and remember their vlaues and adds superflous code. And boolean constants should start with 'IS_'

Reply:
I can rename them.. As far as it being a burden, in a modern IDE like IntelliJ, you can hover over it and it will show you the value without the need to scroll at all. Additionally it's easy to get to the source of that variable via `cmd` -> left click, or right-click -> `Go To` ->  `Declaration or Usages`, or keyboard shortcut `cmd + B`. 

This seems to be a difference in style and shouldn't hold the review. 👍